Library/Formula/go.rb

```diff
@@ -99,5 +120,14 @@ def caveats; <<-EOS.undent
     # This is a a bare minimum of go working as it uses fmt, build, and run.
     system "#{bin}/go", "fmt", "hello.go"
     assert_equal "Hello World\n", `#{bin}/go run hello.go`
+
+    if build.with? "godoc"
+      assert File.exist?(libexec/"bin/godoc")
+      assert File.executable?(libexec/"bin/godoc")
+    end
+    if build.with? "vet"
+      assert File.exist?(libexec/"pkg/tool/darwin_amd64/vet")
+      assert File.executable?(libexec/"pkg/tool/darwin_amd64/vet")
+    end
```

Comment:
No, I was just being paranoid mostly about ensuring they were getting put in the right locations and working as expected long term. I'll try and restrain my paranoia in future :smiley_cat: